Source/WebKit/UIProcess/Cocoa/WebPageProxyCocoa.mm

```diff
@@ -195,12 +195,15 @@ static bool exceedsRenderTreeSizeSizeThreshold(uint64_t thresholdSize, uint64_t
     m_websiteDataStore->loadRecentSearches(name, WTFMove(completionHandler));
 }
 
-void WebPageProxy::grantAccessToCurrentPasteboardData(const String& pasteboardName)
+void WebPageProxy::grantAccessToCurrentPasteboardData(std::optional<FrameIdentifier> frameID, const String& pasteboardName)
```

Comment:
We usually put optional parameter at the end:
`void WebPageProxy::grantAccessToCurrentPasteboardData(const String& pasteboardName, std::optional<FrameIdentifier> = std::nullopt)`